.travis.yml

```diff
@@ -130,7 +130,7 @@ script:
   - go run travis/log_gomaxprocs.go
   - |
     if [[ $TRAVIS_PULL_REQUEST = "false" ]]; then
-      go run test.go $TEST_FLAGS $TEST_MATRIX
+      go run test.go $TEST_FLAGS $TEST_MATRIX -exclude=tests_broken_on_travis
     else
       # Exclude webdriver tests on Travis PRs since the sauce addon does not work for PRs
       go run test.go $TEST_FLAGS $TEST_MATRIX -exclude=webdriver
```

Comment:
oh I think you need to add it here too

Reply:
Yep. Just fixed it. The "if" should have been written as a positive check. That's why I mixed things up.